Update documentation for GISS-GC (#106)
fetch4/GISS-GC: README.md

# GISS-GC

## Description

> This repository contains code as part of an effort in coupling Goddard Institute for Space Studies' [(GISS) Model E](https://simplex.giss.nasa.gov/gcm/) and [GEOS-Chem](https://geoschem.github.io), henceforth GISS-GC. This work was started by Lee Murray in the [GCAP 2.0 project](https://gmd.copernicus.org/articles/14/5789/2021/). The work is part of the [FETCH<sub>4</sub>](https://fetch4.github.io) project. 

Reviewer: ```suggestion
This repository contains code as part of an effort in coupling Goddard Institute for Space Studies' [(GISS) Model E](https://simplex.giss.nasa.gov/gcm/) and [GEOS-Chem](https://geoschem.github.io), i.e., GISS-GC. This work was started by Lee Murray in the [GCAP 2.0 project](https://gmd.copernicus.org/articles/14/5789/2021/). The work is part of the [FETCH<sub>4</sub>](https://fetch4.github.io) project. 
```

> GISS Model E is a general circulation model that can simulate many earth system model configurations, including atmospheric chemistry, aerosols, carbon cycle and other tracers, as well as standard atmosphere, ocean, sea ice and land surface components.

Reviewer: ```suggestion
GISS Model E is a general circulation model (GCM) that can simulate many earth system model configurations, including atmospheric chemistry, aerosols, carbon cycle and other tracers, as well as standard atmosphere, ocean, sea ice and land surface components.
```

GEOS-Chem is a global 3D model of atmospheric chemistry traditionally driven by meteorological input from the Goddard Earth Observing System (GEOS) of the NASA Global Modeling and Assimilation Office.

## Compiling and running the model

> GISS-GC can be built and run on a laptop with a Linux operating system using the following [Linux laptop instructions](https://github.com/fetch4/GISS-GC/blob/develop/doc/GISS-GC-DOC/build_on_linux_laptop.md). A useful tutorial about running the GISS Model E Global Circulation Model (GCM) to drive the GEOS-Chem Chemical Transport Model (CTM) can be found [here](https://github.com/fetch4/GISS-GC/blob/develop/doc/GISS-GC-DOC/giss-gc_tutorial.md).

Reviewer: ```suggestion
GISS-GC can be built and run on a laptop with a Linux operating system using the following [Linux laptop instructions](https://github.com/fetch4/GISS-GC/blob/develop/doc/GISS-GC-DOC/build_on_linux_laptop.md). A useful tutorial about running the GISS Model E GCM to drive the GEOS-Chem Chemical Transport Model (CTM) can be found [here](https://github.com/fetch4/GISS-GC/blob/develop/doc/GISS-GC-DOC/giss-gc_tutorial.md).
```

## Contributing

> Contributions to the code repository are welcome, particularly from anyone interested in using, or implementing this Chemical Transport Model. We welcome addition of details to the model including code.

Reviewer: ```suggestion
Contributions to the code repository are welcome, particularly from anyone interested in using, or implementing this coupled model. We welcome addition of details to the model including code.
```

> Open tickets can be viewed at [Issues](https://github.com/fetch4/GISS-GC/issues).

Reviewer: ```suggestion
Open tickets can be viewed under the [Issues](https://github.com/fetch4/GISS-GC/issues) tab.
```

> To contribute, find a relevant issue or open a new one and assign yourself to work on it. Then create a branch in which to add your contribution and open a pull request. Once ready assign a reviewer and request a code review. Merging should only be performed once a reviewer has approved the changes. 

Reviewer: ```suggestion
To contribute, find a relevant issue or open a new one and assign yourself to work on it. Then create a branch in which to add your contribution and open a pull request. Once ready, assign a reviewer and request a code review. Merging should only be performed once a reviewer has approved the changes. 
```
